site/static/diagrams/canary-testing.puml

```diff
@@ -0,0 +1,35 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+participant Matchmaker
+participant Agones
+
+participant "Canary Fleet\nResource" as CanaryFleet
+participant "Stable Fleet\nResource" as StableFleet
+
+Matchmaker -> Agones: Create: <i>GameServerAllocation</i>
+note left
```

Comment:
The notes on this page are rendered really small and are thus a bit hard to read on the website. I'm not sure there is a way to fix that though.

Reply:
You know what I can do though - I can make the image a link to itself, so if you click it it opens in a new tab/window. I'll do that, so you can relatively easily see the large version.

Reply:
That's a good idea. I got that effect with right click -> open image in new tab but making it a link sounds nice.